src/main/kotlin/no/nav/klage/oppgave/domain/klage/Mottak.kt

```diff
@@ -1,36 +1,58 @@
 package no.nav.klage.oppgave.domain.klage
 
+import no.nav.klage.oppgave.domain.kodeverk.*
+import java.time.LocalDate
 import java.time.LocalDateTime
 import java.util.*
-import javax.persistence.Column
-import javax.persistence.Entity
-import javax.persistence.Id
-import javax.persistence.Table
+import javax.persistence.*
 
 @Entity
 @Table(name = "mottak", schema = "klage")
 class Mottak(
     @Id
     val id: UUID = UUID.randomUUID(),
+    @Column(name = "tema_id")
+    @Convert(converter = TemaConverter::class)
+    val tema: Tema,
+    @Column(name = "sakstype_id")
+    @Convert(converter = SakstypeConverter::class)
+    val sakstype: Sakstype,
     @Column(name = "referanse_id")
-    val referanseId: String,
+    val referanseId: String? = null,
+    @Column(name = "innsyn_url")
+    val innsynUrl: String? = null,
     @Column(name = "foedselsnummer")
-    val foedselsnummer: String,
+    val foedselsnummer: String? = null,
+    @Column(name = "organisasjonsnummer")
+    val organisasjonsnummer: String? = null,
+    @Column(name = "virksomhetsnummer")
+    val virksomhetsnummer: String? = null,
     @Column(name = "hjemmel_liste")
-    val hjemmelListe: String?,
+    val hjemmelListe: String? = null,
+    @Column(name = "avsender_saksbehandlerident")
+    val avsenderSaksbehandlerident: String? = null,
     @Column(name = "avsender_enhet")
-    val avsenderEnhet: Int,
-    @Column(name = "avsender_saksbehandler")
-    val avsenderSaksbehandler: String,
-    @Column(name = "tema")
-    val tema: String,
-    @Column(name = "innsyn_url")
-    val innsynUrl: String? = null,
+    val avsenderEnhet: String? = null,
+    @Column(name = "oversendt_klageinstans_enhet")
+    val oversendtKaEnhet: String? = null,
+    @Column(name = "dato_innsendt")
+    val innsendtDato: LocalDate? = null,
+    @Column(name = "dato_mottatt_foersteinstans")
+    val mottattNavDato: LocalDate? = null,
+    @Column(name = "dato_oversendt_klageinstans")
+    val oversendtKaDato: LocalDate? = null,
+    @Column(name = "dato_frist_fra_foersteinstans")
+    val fristFraFoersteinstans: LocalDate? = null,
     @Column(name = "created")
-    val created: LocalDateTime
+    val created: LocalDateTime = LocalDateTime.now(),
+    @Column(name = "modified")
+    val modified: LocalDateTime = LocalDateTime.now(),
+    @Column(name = "kilde")
+    @Enumerated(EnumType.STRING)
+    val kilde: Kilde
 ) {
 
-    fun hjemler(): List<String> = hjemmelListe?.split(",") ?: listOf()
+    fun hjemler(): List<String> = hjemmelListe?.split(",") ?: emptyList()
```

Comment:
Hadde ikke sett at dette var en kommaseparert liste før nå, men kan jo være at det holder i massevis (i stedet for å legge i egen tabell).

Reply:
Jeg var ikke klar over det selv heller, før jeg begynte på denne PRen. Jeg vurderte å endre det, men droppa det. Tenkte vel omtrent som deg.. :)